sources/packages/backend/workflow/test/2021-2022/parttime-assessment/awards-amounts/parttime-assessment-awards-amount-CSGP.e2e-spec.ts

```diff
@@ -0,0 +1,71 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CSGP.`, () => {
+  it("Should determine federalAwardCSGPAmount when awardEligibilityCSGP is true and offeringCourseLoad is 40 and up", async () => {
```

Comment:
Does `offeringCourseLoad` play any role in this E2E scenario?
I guess that it would be related to `SBSD` not to `CSGP`. Am I missing something?

Reply:
Yeah my bad, it was wrong. Fixed them now.